llvm-general/src/LLVM/General/Internal/FFI/Attribute.h

```diff
@@ -6,9 +6,10 @@
 // parameter attribute, function result attribute or function attribute.
 #define LLVM_GENERAL_FOR_EACH_ATTRIBUTE_KIND(macro)	\
 	macro(None,F,F,F)                                 \
-	macro(Alignment,T,F,F)                            \
+	macro(Alignment,T,T,F)                            \
+	macro(AllocSize,F,F,T)                            \
 	macro(AlwaysInline,F,F,T)                         \
-	macro(ArgMemOnly,T,F,T)                           \
+	macro(ArgMemOnly,F,F,T)                           \
```

Comment:
A general fix, I extracted it in a separate commit to make it easier to cherry pick.